tests/contenttypes_tests/test_models.py

```diff
@@ -256,13 +256,13 @@ def test_message_if_get_for_model_fails(self, mocked_get, mocked_get_or_create):
         """
 
         def _test_message(mocked_method):
+            msg = (
+                "Error creating new content types. Please make sure contenttypes "
+                "is migrated before trying to migrate apps individually."
+            )
             for ExceptionClass in (IntegrityError, OperationalError, ProgrammingError):
                 mocked_method.side_effect = ExceptionClass
-                with self.assertRaisesMessage(
```

Comment:
I think this style is okay as far as flake8 goes... only when the exception is on the same line as assertRaisesMessage and the message is on the next line do we get the warning, "E128 continuation line under-indented for visual indent".